adcm_aio_client/core/types.py

```diff
@@ -21,3 +21,13 @@ class AuthCredentials(NamedTuple):
 AuthToken: TypeAlias = str
 Cert: TypeAlias = str | tuple[str, Optional[str], Optional[str]] | None
 Verify: TypeAlias = str | bool
+
+PathParts: TypeAlias = tuple[str | int, ...]
```

Comment:
```suggestion
type PathParts = tuple[str | int, ...]
```
It's Python 3.12 way